packages/0xcert-scaffold/src/core/gateway.ts

```diff
@@ -309,11 +309,18 @@ export interface AssetLedgerData {
   symbol: string;
 
   /**
-   * Uri base for metadata URI-s. At the end of the base the assetId is automatically appended foo each asset.
+   * Uri base for metadata URI-s. At the end of the base the assetId is automatically appended for each asset.
```

Comment:
* URI base for metadata URIs. At the end of the base, the assetId is automatically appended for each asset.